<launch>
    <node name="heading_controller" pkg="pid" type="controller" output="screen" >
        <param name="Kp" value="1.5" />
        <param name="Ki" value="0.0" />
        <param name="Kd" value="0.0" />
        <param name="upper_limit" value="10" />
        <param name="lower_limit" value="-10" />
        <param name="windup_limit" value="10" />
        <param name="topic_from_plant" value="heading_state" />
        <param name="setpoint_topic" value="heading_setpoint" />
        <param name="topic_from_controller" value="heading_cmd" />
    </node>

    <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure" />
</launch>
